Selenium_Basics2/UnitTest1.cs

```diff
@@ -0,0 +1,161 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+
+
+namespace Selenium_Basics2
+{
+    public class EpumTests
+    {
+        private IWebDriver _chrome;
+
+        private const string MainPage = "https://www.epam.com/";
+
+        private WebDriverWait Waiter { get; set; }
+
+        private TimeSpan _timeout = TimeSpan.FromSeconds(5);
+
+        [SetUp]
+        public void Setup()
+        {
+            _chrome = new ChromeDriver();
+            _chrome.Manage().Window.Maximize();
+            _chrome.Navigate().GoToUrl(MainPage);
+        }
+
+        [SetUp]
+        public void SetupWaiter()
+        {
+            Waiter = new WebDriverWait(_chrome, _timeout);
+            Waiter.IgnoreExceptionTypes(typeof(NoSuchElementException), typeof(ElementNotVisibleException));
+        }
+
+        [Test]
+        public void CheckEpamCareersLocations()
+        {
+            string[] expectedOfficeLocations = { "AMERICAS", "EMEA", "APAC" };
+
+            var action = new Actions(_chrome);
+
+            var careersElement = _chrome.FindElement(By.XPath("//*[@href='/careers']"));
+            action.MoveToElement(careersElement).Build().Perform();
+            careersElement.Click();
+
+            var ourOffices = _chrome.FindElement(By.XPath("//*[@class='tabs__ul js-tabs-links-list']"));
+            Actions scrollDown = new Actions(_chrome);
+            scrollDown.ScrollToElement(ourOffices).Build().Perform();
+
+            var officeElements = ourOffices.FindElements(By.XPath(".//div/a"));
+            Assert.That(officeElements.Count(), Is.EqualTo(3), "Invalid numder of offices");
```

Comment:
do not hardcode numbers in checks. officeElements.Count(), Is.EqualTo(expectedOfficeLocations .Count)